### ISSUE TITLE
Cannot establish secure connections after restoration from TimeMachine

### ISSUE THREAD

**Issue by Zegorax** [AdguardTeam/AdguardForMac#867]
After restoring via TimeMachine, AdGuard doesn't work with all websites. Some new ones that have never been visited before results in "Safari cannot establish a secure connection to ..."

### Steps to reproduce

1. Backup your Mac using TimeMachine with AdGuard installed and running properly
2. Erase your MacOS main disk
3. Restore with TimeMachine
4. AdGuard won't work with some websites

### Expected behavior
AdGuard should work and in the same state prior to restoration

### Actual behavior
AdGuard won't work with some websites. It seems to be the ones that Safari hasn't visited since the restoration while AdGuard is off.

<details><summary>Screenshot:</summary>
![Screenshot](https://user-images.githubusercontent.com/20574959/118086496-5f5fae00-b3c4-11eb-89bc-67604fc67644.png)
</details>

### Customer ID 
Ticket number : 509610

### Your environment
* Environment name and version: (e.g. Chrome 59): Safari  14.1 (16611.1.21.161.6)
* Any specific potentially conflicting software installed: No

**Comment by Chinaski1** [comment]
Hello @Zegorax 

Looks like the issue is not related to the AdGuard app.
Apparently, there was an error in exporting the security certificates when TimeMachine was restored.
You should try to open  the Keychain Access, find and delete [AdGuard Security Certificate](https://uploads.adguard.com/Chinaskiyanqm.png)
- Then open the AdGuard application -> Netowork -> [AdGuard root certificate](https://uploads.adguard.com/Chinaskiy48sk.png) -> Export to any convenient folder
- Open the folder and make double click on AdGuard Personal CA
- KeyChain Access opens, make sure that the certificate marked as [always trust](https://uploads.adguard.com/Chinaskiwkihn.png)

Let us know the result :)

**Comment by Zegorax** [comment]
Yes, I initially suspected the restoration didn't properly restored the root CA. I reinstalled the app and everything was fine. I created the issue because it was an interesting one that may occur to other people in the future.
